main.nf

```diff
@@ -595,10 +527,10 @@ process deepTools {
     file bai from bai_dedup_deepTools.toSortedList()
 
     output:
-    file 'fingerprints.pdf'
-    file 'multiBamSummary.npz'
-    file 'scatterplot_PearsonCorr_multiBamSummary.png'
-    file 'heatmap_SpearmanCorr_multiBamSummary.png'
+    file 'fingerprints.pdf' into deepTools_results_1
```

Comment:
Maybe just this instead of these four lines?
```groovy
file 'fingerprints.pdf multiBamSummary.npz *png' into deepTools_results
```
(not sure if that syntax is correct)